driver/src/main/java/oracle/nosql/driver/query/PlanIter.java

```diff
@@ -157,7 +159,9 @@ static enum FuncCode {
         FN_COUNT_NUMBERS(44),
         FN_SUM(45),
         FN_MIN(47),
-        FN_MAX(48);
+        FN_MAX(48),
+        FN_COLLECT(91),
+        FN_COLLECT_DISTINCT(92);
```

Comment:
Maybe rename to FN_ARRAY_COLLECT/FN_ARRAY_COLLECT_DISTINCT to be consistent with its FuncCode name in KV?

Reply:
Done.